UPGRADE-4.0.md

```diff
@@ -1,6 +1,12 @@
 UPGRADE FROM 3.x to 4.0
 =======================
 
+Debug
+-----
+
+ * `FlattenException::getTrace()` now returns additional type descriptions
+   `integer` and `float`.
+
```

Comment:
Wer only have one changelog Liter component. You can just start a new section by adding a new headline.